angularjs-portal-home/src/main/webapp/my-app/layout/widget/controllers.js

```diff
@@ -399,7 +400,7 @@ define(['angular'], function(angular){
             widgetType : 'rss',
             title: 'RSS Widget',
             jsonSample: false,
-            widgetConfig : {lim : 6, target : '' ,showdate: true, titleLim: 40 , dateFormat: 'MM-dd-yyyy', showShowing: true},
+            widgetConfig : {lim : 6, target : '' ,showdate: true, titleLim: 40 , dateFormat: 'MM-dd-yyyy', showShowing: true, maintenanceMode: false},
```

Comment:
I'm missing something. Maintenance mode flagging is going to come through as `widgetConfig`? How does this relate to flopping apps over into maintenance lifecycle state server-side?